setup.cfg

```diff
@@ -37,7 +37,7 @@ packages = find:
 python_requires = ~=3.7
 install_requires =
   requests>=2.19.1
-  securesystemslib>=0.20.0
+  securesystemslib>=0.22.0
```

Comment:
hmm, this is probably correct but let's make sure we are doing things for right reasons.

It's true that passing the updated test suite requires this... but this is not really an absolute runtime requirement: python-tuf will work with older securesystemslib just as well as it did before this PR. This PR only changes tests  so runtime requirements couldn't possibly have changed: it's a bit weird to imply they did.

On the other hand I don't see an issue with bumping sslib requirement: sslib is practically a part of python-tuf and the bump likely won't make life harder for anyone.

Reply:
I will remove that to escape controversy and give us an option to decide when to bump it.

Reply:
PS: I can update the pr later today. If you still think it's okay to bump the securesystemslib version here then go ahead and merge it.